cmake/modules/HandleLLVMOptions.cmake

```diff
@@ -533,7 +533,7 @@ if(LLVM_USE_SANITIZER)
       message(WARNING "Unsupported value of LLVM_USE_SANITIZER: ${LLVM_USE_SANITIZER}")
     endif()
   else()
-    message(WARNING "LLVM_USE_SANITIZER is not supported on this platform.")
+    append("-fsanitize=address" CMAKE_C_FLAGS CMAKE_CXX_FLAGS)
```

Comment:
I think it would be more appropriate to check the values of LLVM_USE_SANITIZER to make sure that it is set to address as is done for Unix platforms. 

The error message comes from the original LLVM 3.7 source. I'm a bit concerned that they didn't have this enabled either. Do we know anything about what they did to enable it in later versions?

Reply:
Changed into only support Address and report Unsupported value of LLVM_USE_SANITIZER: ${LLVM_USE_SANITIZER}.